src/main/java/io/cdap/plugin/ariba/source/AribaServices.java

```diff
@@ -723,10 +742,13 @@ public boolean isApiLimitExhausted(Response response) {
    * @throws InterruptedException
    */
   @VisibleForTesting
-  void checkAndThrowException(Response response) throws AribaException, InterruptedException {
+  void checkAndThrowException(Response response) throws AribaException, InterruptedException, AribaRetryableException {
     if (response.code() == HttpURLConnection.HTTP_BAD_REQUEST && !AribaUtil.isNullOrEmpty(response.message())) {
       throw new AribaException(response.message(), response.code());
     }
+    if (response.code() != HttpURLConnection.HTTP_OK && AribaUtil.isNullOrEmpty(response.message())) {
```

Comment:
remove this block as at the end it is taken care.

Reply:
Removed !